pkgs/os-specific/darwin/ks/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ks";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "loteoo";
+    repo = pname;
```

Comment:
usage of pname considered anti pattern https://github.com/NixOS/nixpkgs/issues/277994